### PR TITLE
Mini ocr add space

### PR DESCRIPTION
**Pull request by giancarlobi**
We need to add space between words in MiniOCR output to be compliant with new release of Solr plugin highlighting 0.6.0.
This is  a simple way to make this working, probably we can do it better to reflect input hOCR but that needs a whole function rebuild.
This works also with plugin release 0.5.0 without no break in previous release.

### CODE REVIEW

File: src/Plugin/StrawberryRunnersPostProcessor/OcrPostProcessor.php [hunk 1]
```diff
@@ -474,6 +474,7 @@ protected function hOCRtoMiniOCR($output, $pageid) {
         $miniocr->startElement("b");
         $page->registerXPathNamespace('ns', 'http://www.w3.org/1999/xhtml');
         foreach ($page->xpath('.//ns:span[@class="ocr_line"]') as $line) {
+          $notFirstWord = 0;
```
Comment (DiegoPino) [inline src/Plugin/StrawberryRunnersPostProcessor/OcrPostProcessor.php:477]: @giancarlobi simple solution thanks! @pcambra what do you think? maybe make $notFirstWord boolean directly? I have really no preference but may read "easier" if its FALSE, TRUE.

Comment (giancarlobi) [reply src/Plugin/StrawberryRunnersPostProcessor/OcrPostProcessor.php:477]: @DiegoPino I fully agree, boolean seems really better choice

Comment (pcambra) [reply src/Plugin/StrawberryRunnersPostProcessor/OcrPostProcessor.php]: Generally I like booleans better unless it's API/integration stuff

**Comment by DiegoPino** [comment]
I see nothing breaking here so merging. Thanks to both! @giancarlobi when you have a free minute, could you open an issue in Archipelago-deployment that says: upgrade to ocr highlight 0.6 for RC2? Just so we do not forget. Thanks again so much for your follow up and test on this.

**Pull request merged**